_includes/ontology_table.html

```diff
@@ -36,94 +38,78 @@
       </td>
       <td>
         {% if ont.is_obsolete %}
-        <s>
-          {{ ont.description }}
-        </s>
-        <mark>obsolete</mark>
+            <s>
+              {{ ont.description | truncate: 140 }}
+            </s>
+            <mark>obsolete</mark>
         {% elsif ont.activity_status == "inactive" %}
-        {{ ont.description }}
-        <mark>inactive</mark>
+            {{ ont.description | truncate: 140 }}
+            <mark>inactive</mark>
         {% else %}
-        {{ ont.description }}
+            {{ ont.description | truncate: 140 }}
         {% endif %}
         <small>
           <a href="ontology/{{ ont.id }}.html" > Detail </a>
         </small>
       </td>
       <td>
-        <a href="ontology/{{ont.id}}.html">
-          <button type="button" class="btn btn-default" aria-label="Left Align" title="More details">
+        <a href="ontology/{{ont.id}}.html" class="btn btn-default" aria-label="More details for {{ ont.title }}" title="More details">
             <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
-          </button>
         </a>
       </td>
       <td>
-        <a href="{{ont.homepage}}">
-          <button type="button" class="btn btn-default" aria-label="Left Align" title="Project home">
+        <a href="{{ont.homepage}}" class="btn btn-default" aria-label="Go to the homepage for {{ ont.title }}" title="Project home">
```

Comment:
This diff is pretty confusing - wasn't this already address on the main branch? Can we please merge the main branch into this one so the diff makes more sense and is easier to review?

Reply:
I updated from master and diff disappeared. Cheers.

Reply:
Thank you!!